code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -306,7 +303,6 @@
 	name = "medical hardsuit"
 	desc = "A special suit that protects against hazardous, low pressure environments. Built with lightweight materials for easier movement."
 	item_state = "medical_hardsuit"
-	slowdown = 1
```

Comment:
We'll see how this change is received, first. If it trends more positively, then I'll present that in this PR, later.